elasticsearch/examples/kubernetes-kind/values.yaml

```diff
@@ -0,0 +1,37 @@
+---
+# Permit co-located instances for solitary minikube virtual machines.
+antiAffinity: "soft"
+
+# Shrink default JVM heap.
+esJavaOpts: "-Xmx128m -Xms128m"
+
+# Allocate smaller chunks of memory per pod.
+resources:
+  requests:
+    cpu: "100m"
+    memory: "512M"
+  limits:
+    cpu: "1000m"
+    memory: "512M"
+
+# Request smaller persistent volumes.
+volumeClaimTemplate:
+  accessModes: [ "ReadWriteOnce" ]
+  storageClassName: "hostpath"
+  resources:
+    requests:
+      storage: 100M
+extraInitContainers: |
```

Comment:
Could you explain why these extra init containers are needed for a kind deployment? The chart is already correctly setting the permissions and the elasticsearch process should be creating those directories.

Reply:
@Crazybus yeah.. that is needed for KIND.. for some reason the container doesn't create that by default and the whole deployment was failing. You can give it a try and see if you find the reason why the container is failing to do so.. I couldn't see anything in the logs except for an exception saying that it didn't had access to write the files in the `/usr/share/elasticsearch/data/nodes/` directory. Once the directory is created, everything works as expected.

Reply:
Thanks, I'll certainly give it a go to see if I can reproduce it. Seems like a bit of a shame that kind doesn't seem to be compatible with the way that normal kubernetes works though :(. It makes sense that resources and node anti affinity need to be relaxed but this makes it impossible to properly test with kind.

Reply:
It should be super straight forward to reproduce it and find the cause if you can debug the ElasticSearch container. But for now that is needed.

Reply:
@Crazybus also I notice that for KIND this is required:  `storageClassName: "standard"`

Reply:
Sorry that I haven't gotten around to testing this out just yet. Did you also try just leaving `storageClassName` out completely? That should just use whatever the default is set to which is probably "standard"

Reply:
that is true.. I am removing that line completely